scout/models/disease_term.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional, Union
+
+from pydantic import BaseModel, field_validator
+
+
+class DiseaseTerm(BaseModel):
```

Comment:
💯
Good stuff! But shouldn't this be in like a general `models/loading_models.py`, in analogy with the  
`models.case.case_loading_models` and `models.case.case`?

Not needed, but it would make it clearer to me to move the old class DiseaseTerm, imported from `models.phenotype_term` into a `models.disease_term`.

Reply:
There is a difference between a case model (we don't have it in place yet) and a case loading model. After loading a case, you can set several key/values that are missing in the loading model..

But the disease term is pretty much the same. I think it's fine to leave it like this

Reply:
Ah, but we do: https://github.com/Clinical-Genomics/scout/blob/8d212fd0863c8c22130ea3db2ec583d66ca32ee9/scout/models/phenotype_term.py#L34
I think I will loose track if the disease load one is in a file of its own, and the db data model one together with the phenotypes.

Reply:
But, I can drop an issue on it instead anyway and we can do that separately - it doesn't have to be now.